exercises/java/src/main/java/com/adaptionsoft/games/board/Board.java

```diff
@@ -0,0 +1,62 @@
+package com.adaptionsoft.games.board;
```

Comment:
Just for my understanding, this is a rewrite that if you had more time you'd pull the logic into then clear up the game class?

Reply:
Yeah exactly. I was going to hook it up into the game, then clean up the game to be smaller but everything failed so I left it :/